### PR TITLE
Add :any-link selector data

### PR DESCRIPTION
**Pull request by ai**
I take Mozilla data from [MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link). For webkit I found first question on Stack Overflow and use this data as start point.

### CODE REVIEW

**Comment by ai** [comment]
@Fyrd this PR is important for Autoprefixer https://github.com/postcss/autoprefixer/issues/692

File: features-json/css-any-link.json [hunk 1]
```diff
@@ -0,0 +1,282 @@
+{
+  "title":"CSS Filter Effects",
```
Comment (cvrebert) [inline features-json/css-any-link.json]: Looks like you didn't update the title, description, etc. fields.

Comment (ai) [inline features-json/css-any-link.json]: Oops 😞 will fix tomorrow Monday, sorry

**Comment by ai** [comment]
@cvrebert sorry, it was git issue, I commited non-changed file :'( shame on me. Fixed right now.

File: features-json/css-any-link.json [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+      "41":"n",
+      "42":"n",
+      "43":"n",
+      "44":"n",
+      "45":"n",
+      "46":"n",
+      "47":"n",
+      "48":"n",
+      "49":"n",
+      "50":"y x"
```
Comment (cvrebert) [inline features-json/css-any-link.json]: Erm, they removed the prefix in Firefox 50, so this ought to be `"y"`.

Presumably there should be a long run of `"y x"` for versions prior to 50. Obviously someone needs to test exactly how far back the prefixed support goes.

**Comment by ai** [comment]
@cvrebert updated. I found in [issue about unprefixing](https://bugzilla.mozilla.org/show_bug.cgi?id=843579) that at least in 2013 they have this prefix.

**Comment by cvrebert** [comment]
This addresses issue #1742.

**Pull request merged**